bundles/org.openhab.binding.homeconnect/src/main/java/org/openhab/binding/homeconnect/internal/handler/AbstractHomeConnectThingHandler.java

```diff
@@ -1006,27 +1008,39 @@ protected ChannelUpdateHandler defaultAmbientLightChannelUpdateHandler() {
-                                .ifPresent(channel -> updateState(channel.getUID(),
-                                        new PercentType(brightnessData.getValueAsInt())));
+                        getThingChannel(CHANNEL_AMBIENT_LIGHT_BRIGHTNESS_STATE).ifPresent(channel -> {
+                            try {
+                                Data brightnessData = apiClient.get().getAmbientLightBrightnessState(getThingHaId());
+                                updateState(channel.getUID(), new PercentType(brightnessData.getValueAsInt()));
+                            } catch (CommunicationException | ApplianceOfflineException | AuthorizationException e) {
+                                updateState(channel.getUID(), UnDefType.UNDEF);
+                            }
+                        });
 
                         // color
-                        Data colorData = apiClient.get().getAmbientLightColorState(getThingHaId());
-                        getThingChannel(CHANNEL_AMBIENT_LIGHT_COLOR_STATE).ifPresent(
-                                channel -> updateState(channel.getUID(), new StringType(colorData.getValue())));
+                        getThingChannel(CHANNEL_AMBIENT_LIGHT_COLOR_STATE).ifPresent(channel -> {
+                            try {
+                                Data colorData = apiClient.get().getAmbientLightColorState(getThingHaId());
+                                updateState(channel.getUID(), new StringType(colorData.getValue()));
+                            } catch (CommunicationException | ApplianceOfflineException | AuthorizationException e) {
+                                updateState(channel.getUID(), UnDefType.UNDEF);
+                            }
+                        });
 
                         // custom color
-                        Data customColorData = apiClient.get().getAmbientLightCustomColorState(getThingHaId());
                         getThingChannel(CHANNEL_AMBIENT_LIGHT_CUSTOM_COLOR_STATE).ifPresent(channel -> {
-                            String value = customColorData.getValue();
-                            if (value != null) {
-                                updateState(channel.getUID(), mapColor(value));
-                            } else {
+                            try {
+                                Data customColorData = apiClient.get().getAmbientLightCustomColorState(getThingHaId());
+                                String value = customColorData.getValue();
+                                if (value != null) {
+                                    updateState(channel.getUID(), mapColor(value));
+                                } else {
+                                    updateState(channel.getUID(), UnDefType.UNDEF);
+                                }
+                            } catch (CommunicationException | ApplianceOfflineException | AuthorizationException e) {
                                 updateState(channel.getUID(), UnDefType.UNDEF);
```

Comment:
Shouldn't you set the Thing status to OFFLINE here instead?

Reply:
Undone this change